src/CopyrighFixer/CopyrighFixer/owner.cpp

```diff
@@ -20,12 +20,38 @@ const QString &Owner::getOwnerName() const {
     return _name;
 }
 
-void Owner::setTimeRange(const QString &interval) {
-    _timeRange = interval;
+void Owner::setTimePoint(const int &interval) {
+    _timePoint = interval;
 }
 
-const QString &Owner::getTimeRange() const {
-    return _timeRange;
+const int &Owner::getTimePoint() const {
```

Comment:
replace const int& to int
```suggestion
int Owner::getTimePoint() const {
```